nemo/collections/nlp/nm/trainables/common/common_utils.py

```diff
@@ -31,7 +31,7 @@ def get_pretrained_lm_models_list():
     return get_megatron_lm_models_list() + get_huggingface_lm_models_list()
 
 
-def get_pretrained_lm_model(pretrained_model_name, config=None, vocab=None, checkpoint=None):
+def get_pretrained_lm_model(pretrained_model_name, config=None, vocab=None, checkpoint=None, local_rank=None):
```

Comment:
could you please update the docstring with the new arg

Reply:
Done.